README.md

```diff
@@ -1,13 +1,20 @@
 This project is a build of my software develper portfolio. It is also a test of transferring FIGMA designs into
```

Comment:
- [ ] REQUIRED: As stated by the previous reviewer, let's kindly use [this Microverse template](https://github.com/microverseinc/readme-template) in your readme.md and customize it with the information pertaining to your project.  

Please use this same branch to do it, so we can review it.
This template will serve you well in your future journey in Microverse for all your projects because it contains the minimum required items for a repo to be descriptive and professional.
(You are free to improve it with more items and/or decorations).
Please, kindly remember to fill the template with this project's info.

That way, your repo will look professional! 💪